README.md

```diff
@@ -235,6 +235,17 @@ To implement new checks and test them:
 - locate the newly created Check in the Catalog directory `./priv/catalog/`
 - test the execution as [previously described](#testing-executions)
 
+# Installing a development environment
+
+To set up a local development environment for hacking on Wanda, you need to follow the instructions provided in [hack on wanda](./guides/development/hack_on_wanda.md).
```

Comment:
Let's use absolute links so we do not have issues when the doc is rendered at https://www.trento-project.io/wanda/

Reply:
Yeah you are right, i was focused on the github repo , good catch!

Reply:
Looking at it better, maybe a relative link here is good, so we'll be redirected to the relevant doc page since we have registered it in `mix.exs`